bin/internal/dart-sdk.version

```diff
@@ -1 +1 @@
-1.25.0-dev.7.0
+hash/a83a69ff3eff845980308779cadbdb80cc03a603
```

Comment:
It would be nice to be disentangled from rolls of the sdk into the internal Google's repo.

Would it be worth for some of us to meet up to brainstorm ideas about this?

I like @zanderso suggestion to make the API stable, but there is some nuance we need to think more about. For example, we need flutter-engine and flutter-tools to depend on the same version so that the binary format in package:kernel and the vm deserializer match (unless we add backwards compatibility to the format)